frontends/web/src/components/view/view.module.css

```diff
@@ -75,7 +75,7 @@
     }
     .inner.fit {
         padding-bottom: var(--space-half);
-        padding-top: 2vh;
+        padding-top: 0;
```

Comment:
@shonsirsha  moved this together with another small fix to separate small PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/2612